app/assets/javascripts/markerclusterer.js

```diff
@@ -0,0 +1,1306 @@
+
```

Comment:
Put this in vendor/assets/javascripts and explicitly require it in the application.js